### PR TITLE
Launch feedback form to external browser

### PR DESCRIPTION
**Pull request by suesitran**
☝️ Related to https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues/46

When user clicks on App Feedback in Information View, open Feedback Form in external browser

[*] `release-notes.txt` has been updated.

## Changes

- Add Feedback form URL https://forms.gle/zQtfhvswrKmjJjNV7
- Add fallback action in App Feedback card

### CODE REVIEW

File: ios/Podfile.lock
Comment (hades-6) [inline ios/Podfile.lock:109]: why the downgrade?

**Review by hades-6 — APPROVED** [review]

File: ios/Podfile.lock
Comment (suesitran) [reply ios/Podfile.lock:109]: that's my version of cocoapod in my system. This file is auto generated when u run `flutter packages get`.

**Comment by lukesleeman** [comment]
👋 @suesitran I've requested all the flutter devs to review, hopefully that gets you some more 👀 

I see @hades-6 has approved, so if your still stuck on this after a few hours, and want me to approve, just message me.  I'll be happy to do it. 👍

File: ios/Podfile.lock
Comment (ericjiang97) [reply ios/Podfile.lock:109]: What version of flutter are you running? 

I think we should be on the beta channel for flutter atm.

Comment (rockgecko-development) [reply ios/Podfile.lock:109]: @ericjiang97 This is the Podfile.lock file, not Pubspec.yaml... Flutter version is not relevant :)

**Review by rockgecko-development — APPROVED** [review]
LGTM

**Pull request merged**
